assets/js/hooks/index.ts

```diff
@@ -384,23 +384,118 @@ export const OpenAuthorizeUrl = {
   },
 } as PhoenixHook;
 
-export const EditScope = {
+export const TagInput = {
   mounted() {
-    this.el.addEventListener('dblclick', _e => {
-      const scopeValue = this.el.dataset.scope;
-      const eventType = this.el.dataset.eventType;
-      this.pushEventTo(this.el, eventType, { scope: scopeValue });
+    this.container = this.el;
+    this.textInput = this.el.querySelector('input[type="text"]');
+    this.hiddenInput = this.el.querySelector('input[type="hidden"]');
```

Comment:
can we just have these explicitly specified in the hook definition? 
`<div phx-hook="TagInput" data-text-el={el-id} data-hidden-el={hidden-el-id} data-tag-list={el-id}>`

Reply:
Also, we can raise an exception when these elements aren't found

Reply:
Done, thanks